addon/components/cfb-form-editor/general.js

```diff
@@ -88,28 +88,33 @@ export default Component.extend(ComponentQueryManager, {
     }
   }).drop(),
 
-  delete: task(function*(changeset) {
+  archive: task(function*(changeset) {
     try {
-      yield this.get("apollo").mutate({
-        mutation: deleteFormMutation,
-        variables: {
-          input: {
-            id: changeset.get("id"),
-            clientMutationId: v4()
+      let form = yield this.get("apollo").mutate(
```

Comment:
Why not `const`?

Reply:
Fair point, changed!

Reply:
Might be a good idea to enable eslint's "prefer const" rule